### PR TITLE
{tools}[GCCcore/12.1.0] cURL v7.84.0

### PR DESCRIPTION
**Pull request by absrocks**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/c/cURL/cURL-7.84.0-GCCcore-12.1.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ form based upload, proxies, cookies, user+password authentication (Basic,
+ Digest, NTLM, Negotiate, Kerberos), file transfer resume, http proxy tunneling
+ and more.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.1.0'}
+
+source_urls = ['https://curl.haxx.se/download/']
+sources = [SOURCELOWER_TAR_GZ]
+#checksums = ['c0e64302a33d2fb79e0fc4e674260a22941e92ee2f11b894bf94d32b8f5531af']
```
Comment (akesandgren) [inline easybuild/easyconfigs/c/cURL/cURL-7.84.0-GCCcore-12.1.0.eb:22]: checksum must not be commented out.

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/cURL/cURL-7.84.0-GCCcore-12.1.0.eb (hunk 1 above)
Comment (absrocks) [reply easybuild/easyconfigs/c/cURL/cURL-7.84.0-GCCcore-12.1.0.eb]: I added the checksum verification

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Review by akesandgren — APPROVED** [review]

**Comment by akesandgren** [comment]
Going in, thanks @absrocks!

**Comment by absrocks** [comment]
Great! Thanks @akesandgren!

**Pull request merged**